core/modules/system/system.install

```diff
@@ -2814,6 +2827,13 @@ function system_update_1056() {
 }
 
 /**
+ * Change default state for display error messages.
+ */
+function system_update_1057() {
+  config_set('system.core', 'error_level', 'hide');
```

Comment:
Yep, we can change the default on new sites but we shouldn't be changing the configuration of existing ones.